packages/wallet/src/redux/protocols/concluding/__tests__/reducer.test.ts

```diff
@@ -24,11 +24,11 @@ describe('[ Happy path ] scenario', () => {
     const action = scenario.actions.concludeReceived;
```

Comment:
This action needs to be wired up to somewhere else in the app (the message listener saga?)